src/LoginContainer/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import './style.css';
+import { loginUser } from '../actions/user';
+
+
```

Comment:
File name can be LoginUserContainer or just LoginUser.
Actually container means you're just connecting to redux and passing to an another component.
But here you're directly connecting to a component and using it here itself. So you can avoid the name `container`